openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -294,9 +294,9 @@ def add_staff_markup(user, disable_staff_debug_info, block, view, frag, context)
         histogram = None
         render_histogram = False
 
-    if settings.FEATURES.get('ENABLE_LMS_MIGRATION') and hasattr(block.runtime, 'filestore'):
+    osfs = block.runtime.resources_fs
+    if settings.FEATURES.get('ENABLE_LMS_MIGRATION') and osfs:
```

Comment:
Just wanted to confirm, whether it's okay to remove `hasattr` check here?

Reply:
Thank you for the review, @DubeySandeep. I think you're right, it's safer to leave `hasattr` in place. I added it back.